purchases/src/main/kotlin/com/revenuecat/purchases/common/networking/HTTPResult.kt

```diff
@@ -52,7 +54,20 @@ internal data class HTTPResult(
             } else {
                 null
             }
-            return HTTPResult(responseCode, payload, origin, requestDate, verificationResult, isLoadShedderResponse)
+            val isFallbackURL: Boolean? = if (jsonObject.has(SERIALIZATION_NAME_IS_FALLBACK_URL)) {
+                jsonObject.getBoolean(SERIALIZATION_NAME_IS_FALLBACK_URL)
+            } else {
+                null
```

Comment:
Related to my previous comment: and this would be just `false`. Mostly I'm afraid that we have `false` and `null` mean the same here